test/performance/locustfile.py

```diff
@@ -0,0 +1,76 @@
+import urllib3, os, json
+from datetime import datetime
+from string import Template
+from uuid import uuid4
+from pathlib import Path
+from locust import HttpUser, task
+from cloudevents.http import CloudEvent
+from cloudevents.conversion import to_binary
+
+urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
+
+token = os.getenv("SA_TOKEN", "")
+if token == "":
+    raise Exception("SA_TOKEN environment variable is empty")
+
+with open(Path(__file__).parent / "pod.json") as fd:
+    template = Template(fd.read())
+
+class GetPods(HttpUser):
```

Comment:
Double blank line before a class is [recommended by PEP8](https://peps.python.org/pep-0008/#blank-lines)

```suggestion


class GetPods(HttpUser):
```

Reply:
Done